lib/web_ui/lib/src/engine/view_embedder/embedding_strategy/full_page_embedding_strategy.dart

```diff
@@ -5,6 +5,7 @@
 import 'package:ui/src/engine/dom.dart';
 import 'package:ui/src/engine/util.dart' show setElementStyle;
 
+import '../hot_restart_cache_handler.dart';
```

Comment:
~~Unused import?~~ No, it's a free function!

```suggestion
import '../hot_restart_cache_handler.dart' show registerElementForCleanup;
```